src/stdlib/contains_all.rs

```diff
@@ -0,0 +1,152 @@
+use crate::compiler::prelude::*;
+use crate::stdlib::string_utils::convert_to_string;
+
+fn contains_all(value: Value, substrings: Value, case_sensitive: bool) -> Resolved {
```

Comment:
Is it worth making this function a bit more generic? I could see someone asking for a `contains_any` for the same reasons as `contains_all`.

Reply:
We could also look into using https://crates.io/crates/aho-corasick for a significant perf boost when used with long strings. That algorithm is specifically designed for this.

Reply:
Yes, good point. This is what inspired: https://github.com/vectordotdev/vrl/issues/458. We could make this more generic by adding OR/AND semantics.

For the perf work, I generally like to do this as a secondary step. I will look into it.

Reply:
Discussed offline: 

- keeping this as is
- we can add `contains_any` in the future if there's demand
  - `aho-corasick` seems a good candidate there